packages/components/src/submenu-popover/index.tsx

```diff
@@ -81,9 +83,9 @@ function useHasRightSpace( parentElement: HTMLElement | undefined, isVisible: bo
 }
 
 export function useSubmenuPopoverProps< T extends HTMLElement >(
-	options: { offset?: number } = {}
+	options: { offset?: 0; flip?: true; resize?: true; inline?: false } = {}
 ) {
-	const { offset = 0 } = options;
+	const { offset = 0, inline = false, flip = true, resize = true } = options;
```

Comment:
```suggestion
export function useSubmenuPopoverProps< T extends HTMLElement >(
	options: Omit< SubmenuPopoverProps, 'isVisible' | 'placement' | 'anchor' | 'children' > = {
		offset: 0,
		flip: true,
		resize: true,
		inline: false,
	}
) {
	const { offset, inline, flip, resize } = options;
```
Actually I was thinking something like that

Reply:
The reason is that the props we are passing are actually a subset of what submenu props are ( `const submenu: Partial< SubmenuPopoverProps >` ). If for some reason `SubmenuPopoverProps` changes completely, the suggestion I made will create a ts error. However, this was not the case before, as there was not any relationship between options and SubmenuPopoverProps ( which it should have ).